x/ibc/04-channel/client/cli/tx.go

```diff
@@ -0,0 +1,393 @@
+package cli
```

Comment:
The handshake command is implemented in this PR https://github.com/cosmos/cosmos-sdk/pull/5124 we should port the implementation from there. 

Also looks like each individual transaction needs to be implemented separately a la ICS-3 implementation.